plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1350,7 +1350,11 @@ public function disable_compat_mode(): void {
 	 */
 	private function toggle_compat_mode( bool $enabled ): void {
 		if ( ! $this->synchronizer->check_orders_table_exists() ) {
-			WP_CLI::error( __( 'HPOS tables do not exist.', 'woocommerce' ) );
+			if ( $enabled ) {
+				$this->synchronizer->create_database_tables();
+			} else {
+				WP_CLI::error( __( 'HPOS tables do not exist. ' . $create_tables, 'woocommerce' ) );
```

Comment:
Hey @Mayisha! I think this `$create_tables` variable is not defined.

Reply:
Thanks @albarin 👍 
My bad! I missed to cleanup this part from an alternate approach. Fixed in [51e6e8b](https://github.com/woocommerce/woocommerce/pull/51973/commits/51e6e8bc5c2952e950e47950c31f7f90959011f0)